### ISSUE TITLE
You should add instruction

### ISSUE THREAD

**Issue by Nicklehert** [jameshnsears/QuoteUnquote#147]
What's CSV file ?
Where can I download the CSV file for the quotes ?
How do I link the file to the widget ?
Is there a youtube video explaining how to do it ??

There are a lot of things on the widget that could mean literally anything. Have I spent the last 3 hours trying to figure this out

So on and so forth. The below average joe such as myself would find it difficult to understand how to do things. ( There is a reason why they have instructions. What is commonsense for some people may not be for others) 
I learnt what CSV is by trying to figure out how to get the app widget to work (it still hasn't worked because Idk where to get the quotes from)

**Comment by jameshnsears** [comment]
Hi - thanks for the feedback, it is helpful and I do really appreciate it.

I apologise for the situation you found yourself in.

Going forward, I'll expand the instructions page to mention .csv usage; I'll also publish a YouTube video that shows how to install the app and get to the quotations as quickly as possible. I'll do both these things in the next release - 4.27.0 - which I'll try and publish with the next couple of days.

Speaking generally, the app comes supplied with around 20,000 built in quotations, in the "Internal" database. A .csv is only required if you want import your own quotations into the "External" database. 

If after following the current instructions - that come supplied with the app and are visible when you click on the Quote Unquote app icon - your still having a problem viewing the quotations then update this issue with what you did and I'll see how I can help more.

Once again, thanks for taking the time to let me know how to improve the app experience.

**Comment by jameshnsears** [comment]
I've release 4.27.0 - it might take a day or so to appear in Google Play Store / F-Droid.

I published a simple video on Google Play showing how to install and quickly use the App Widget, without having to worry about any .csv file.

I've also updated the instructions and the Configuration screen to try to improve things.

I'll close this issue now but don't hesitate to contact me if you hit a problem.
